app/Filament/Resources/ServerResource/Pages/CreateServer.php

```diff
@@ -520,6 +353,70 @@ public function form(Form $form): Form
                                         ->columnSpan(2),
                                 ]),
                         ]),
+
+                    Wizard\Step::make('Allocation')
+                        ->label('Allocation')
+                        ->icon('tabler-transfer-in')
+                        ->completedIcon('tabler-check')
+                        ->columns(4)
+                        ->schema([
+
+                            Forms\Components\TagsInput::make('ports')
```

Comment:
I added a port that already exists (`25565`). (this shouldn't be possible in the first place)
I can't set it as `SERVER_PORT` but `QUERY_PORT` is fine apparently:
![grafik](https://github.com/user-attachments/assets/fd21546f-9c55-4acc-b3e9-f491ca8e0ba2)